test/util/test_runners/integrations_tests_runner.py

```diff
@@ -0,0 +1,136 @@
+"""
+This is a runner module for all integration tests.
+The purpose of this file is to be a cross platform tests runner
+for all integration tests
+
+Roadmap:
+1) run all integration unitests from this file
+2) set up the system env to run the unitests
+"""
+
+# biultIn
+import sys
+import os
+
+# Third party
+import unittest
+import base64
+import requests
+from pathlib import Path
+import logging
+import subprocess as sb
+import shutil
+
+# internals
+from test.test_cli_integration import CliIntegrationTest, CliIntegrationTestConfigurations
+from conjur.endpoints import ConjurEndpoint
+from test.util.test_runners.integration_test_case import IntegratioTestCaseBase
+from test.util.test_runners.test_runner_args import TestRunnerArgs
+from test.util.test_runners.params import ClientParams, TestEnvironmentParams
+from test.util.test_path_provider import TestRunnerPathProvider
+
+
+class TestRunner:
+
+    def __init__(self, args: TestRunnerArgs):
+        self.runner_args = args
+        path_provider = TestRunnerPathProvider(helpers_file_dir=args.files_folder)
+        self.env_params = TestEnvironmentParams(args.cli_to_test_path, args.invoke_cli_as_process, path_provider)
+        self.client_params = ClientParams(self.runner_args.hostname, self.runner_args.username,
+                                          self.runner_args.account)
+
+    def run_tests(self, test_modules: list):
+        """
+        @param test_modules: the tests classes wanted to run
+        """
+        res = []
+        for test_module in test_modules:
+            res.append(self.__run_test_cases(test_module))
+        sys.exit(not all(res))
+
+    def start_env(self):
+        """
+        set up the test environment
+        Test environment setup will be in the following order:
+        1) Create a netrc file to provide login details
+        2) Run the CLI to initialize the CLI to get the certificate and conjurrc files
+        """
+
+        # copy netrc from home folder to test folder
```

Comment:
we can create the netrc before hand. We need to generate it on runtime because the api key will change. this can be a temporary solution until we merge my PR then we can just use the `conjur login` command to do it for us

Reply:
![wait_for_it](https://user-images.githubusercontent.com/76062747/103552173-282c3e00-4eb4-11eb-924a-94864eda84d9.gif)
can't wait